src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -26,6 +27,10 @@ def __init__(self, **kwargs):
         #this will break all in one test
         self.reqMgr2 = ReqMgr(kwargs.get("reqmgr2_endpoint", None))
         self.previous_state = {}
+        localurl = kwargs.get('logdb_url', '')
+        identifier = kwargs.get('hostName', '')
+        centralurl = kwargs.get('central_logdb_url', '')
+        self.logdb = LogDB(localurl, identifier, centralurl, logger=self.logger)
```

Comment:
since ther is no localurl, shouldn't this be 
self.logdb = LogDB(centralurl, identifier, None, logger=self.logger)